modules/text2vec-ollama/module.go

```diff
@@ -26,18 +26,30 @@ import (
 	"github.com/weaviate/weaviate/entities/modulecapabilities"
 	"github.com/weaviate/weaviate/entities/moduletools"
 	"github.com/weaviate/weaviate/modules/text2vec-ollama/clients"
-	"github.com/weaviate/weaviate/modules/text2vec-ollama/vectorizer"
+	"github.com/weaviate/weaviate/modules/text2vec-ollama/ent"
 	"github.com/weaviate/weaviate/usecases/modulecomponents/additional"
 )
 
 const Name = "text2vec-ollama"
 
+var batchSettings = batch.Settings{
+	// the encoding is different than OpenAI, but the code is not available in Go and too complicated to port.
+	// using 30% more than the OpenAI model is a rough estimate but seems to work
+	TokenMultiplier:    1.3,
+	MaxTimePerBatch:    float64(10),
+	MaxObjectsPerBatch: 512, // Info from jina
+	// real limit is 8192, but the vectorization times go up by A LOT if the batches are larger
+	MaxTokensPerBatch: func(cfg moduletools.ClassConfig) int { return 2500 },
+	HasTokenLimit:     true,
+	ReturnsRateLimit:  false,
+}
```

Comment:
these seem to be copied from the jina module.

I think:
- TokenMultiplier: 0
- MaxObjectsPerBatch ????
- MaxTokensPerBatch High dummy value. If the multiplier is 0 any number above 0 should be fine though

Reply:
pushed a proposal too but would like to have a review if proposed values are ok

Reply:
I think we should do HasTokenLimit:  false,